scrapy/utils/misc.py

```diff
@@ -39,10 +39,19 @@ def arg_to_iter(arg):
 def load_object(path):
     """Load an object given its absolute object path, and return it.
 
-    object can be the import path of a class, function, variable or an
-    instance, e.g. 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware'
+    The object can be the import path of a class, function, variable or an
+    instance, e.g. 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware'.
+
+    If ``path`` is not a string, but a callable() object, then return it as is.
```

Comment:
What do you think about mentioning that classes are callable? I think this is a rather arcane knowledge, but it is the main use case here.